webapp/components/survey/Record/store/useLocalState.js

```diff
@@ -59,7 +59,9 @@ export const useLocalState = () => {
     }
 
     // Reset form
-    dispatch(SurveyFormActions.resetForm())
+    if (!preview) {
+      dispatch(SurveyFormActions.resetForm())
```

Comment:
can you check if this is needed? provably not ?

Reply:
cleaned :)